crates/rome_js_formatter/src/js/expressions/static_member_expression.rs

```diff
@@ -1,55 +1,105 @@
-use rome_js_syntax::JsSyntaxKind;
-use rome_rowan::AstNode;
-
 use crate::prelude::*;
-
 use crate::FormatNodeFields;
-use rome_js_syntax::JsStaticMemberExpression;
-use rome_js_syntax::JsStaticMemberExpressionFields;
+use rome_formatter::ConcatBuilder;
+use rome_js_syntax::{
+    JsStaticMemberExpression, JsStaticMemberExpressionFields, JsSyntaxKind, JsSyntaxNode,
+    JsSyntaxToken,
+};
+use rome_rowan::AstNode;
 
 impl FormatNodeFields<JsStaticMemberExpression> for FormatNodeRule<JsStaticMemberExpression> {
     fn format_fields(
         node: &JsStaticMemberExpression,
         formatter: &JsFormatter,
     ) -> FormatResult<FormatElement> {
+        let mut current = node.clone();
+
+        while let Some(static_member_expression) =
+            JsStaticMemberExpression::cast(current.object()?.syntax().clone())
```

Comment:
Is it possible to use `into_syntax` to avoid the clone?

```suggestion
            JsStaticMemberExpression::cast(current.object()?.into_syntax())
```